EventSite/Views/Events/CreateOrUpdate.cshtml

```diff
@@ -42,18 +43,30 @@
                                 @Html.ValidationMessage("Event.Name")
                             </div>
                         </div>
-                    
+
+
                         <div class="form-group">
                             <label for="Slug" class="col-lg-2 control-label">Slug</label>
                             <div class="col-lg-10">
-                                @if(isCreate) {
+                                @if (isCreate)
+                                {
                                     <input type="text" class="form-control" id="Slug" name="Slug" placeholder="Slug" value="@Model.Slug" />
                                     @Html.ValidationMessageFor(m => m.Slug)
-                                } else {
-                                    <p class="form-control-static">@Model.Slug</p>
+                                }
+
+                                @if (!isCreate)
```

Comment:
the else that you dropped reads a lot better than two different if statements back to back.